databricks/koalas/tests/test_rolling.py

```diff
@@ -74,3 +74,40 @@ def test_rolling_sum(self):
 
     def test_rolling_count(self):
         self._test_rolling_func("count")
+
+    def _test_groupby_rolling_func(self, f):
+        kser = ks.Series([1, 2, 3])
+        pser = kser.to_pandas()
+        self.assert_eq(
+            repr(getattr(kser.groupby(kser).rolling(2), f)().sort_index()),
+            repr(getattr(pser.groupby(pser).rolling(2), f)()))
+
+        # Multiindex
+        kser = ks.Series(
+            [1, 2, 3],
+            index=pd.MultiIndex.from_tuples([('a', 'x'), ('a', 'y'), ('b', 'z')]))
+        pser = kser.to_pandas()
+        self.assert_eq(
+            repr(getattr(kser.groupby(kser).rolling(2), f)().sort_index()),
+            repr(getattr(pser.groupby(pser).rolling(2), f)()))
+
+        kdf = ks.DataFrame({'a': [1, 2, 3, 2], 'b': [4.0, 2.0, 3.0, 1.0]})
```

Comment:
Let's dont forgot to add multi column test

Reply:
okay. i've wrote down my notepad and never forget 😄